src/redux/reducers/componentConfig/initialState.js

```diff
@@ -139,6 +139,7 @@ const frequencyInitialConfig = {
   frequencyType: 'proportional',
   metadata: 'condition',
   legendEnabled: true,
+  metadataDisabled: false,
```

Comment:
Can you tell me what is this property for?

Reply:
Not needed, just forgot to remove it.